packages/server/src/games.ts

```diff
@@ -193,11 +195,43 @@ export async function handleMoveAndTime(
 
   game.moves.push(moves);
 
-  io().emit(`game/${game_id}`, game);
+  emitGame(game.id, game.players?.length ?? 0, game_obj);
 
   return game;
 }
 
+function emitGame(
+  game_id: string,
+  num_players: number,
+  game_obj: AbstractGame<unknown, unknown>,
+): void {
+  const next_to_play = game_obj.nextToPlay();
+  const specialMoves = game_obj.specialMoves();
+
+  io()
+    .to(`game/${game_id}`)
+    .emit("move", {
+      state: game_obj.exportState(null),
+      round: game_obj.round,
+      next_to_play: next_to_play,
+      special_moves: specialMoves,
+      result: game_obj.result,
+      seat: null,
+    });
+
+  for (let seat = 0; seat < num_players; seat++) {
+    const gameStateResponse: GameStateResponse = {
+      state: game_obj.exportState(seat),
+      round: game_obj.round,
+      next_to_play: next_to_play,
+      special_moves: specialMoves,
+      result: game_obj.result,
+      seat: seat,
+    };
+    io().to(`game/${game_id}/${seat}`).emit("move", gameStateResponse);
```

Comment:
Does this get used in the client?  Seems like it should, but I didn't see any `emit("subscribe", [`game/${game_id}/${seat}`])`

Reply:
Yes the subscription happens in the function setPlayingAs in GameView.vue

Reply:
Totally missed that! Thanks!